libs/ui-watt/src/lib/components/checkbox/watt-checkbox.component.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+watt-checkbox {
+  --watt-checkbox-disabled-color: var(--watt-color-neutral-grey-400);
+  --watt-checkbox-color: var(--watt-color-primary);
+  --watt-checkbox-checkmark-color: var(--watt-on-dark-high-emphasis);
+
+  display: block;
+
+  input {
+    appearance: none;
+    margin: 0;
+    padding: 0;
+    width: 1.15em;
+    height: 1.15em;
+    border: 0.15em solid var(--watt-checkbox-color);
+    display: grid;
+    place-content: center;
+  }
```

Comment:
issue: the new checkbox is a bit square-ish compared to Material (and in Figma).

Let's round the corners a bit (2px I believe) and make the checkmark a bit thinner.

New / Current
![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/196bc224-fef6-407a-b8e7-f26867450143)

Design in Figma ([link](https://www.figma.com/file/fwEDQkgEcpwfUojXl0EIC5/Watt-Design-System?type=design&node-id=4761-12635&mode=design&t=vcMKrq08ouepUVuD-0)) 

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/46c476c0-fb77-4787-a9bf-4dc2bf714625)